priorityq.py

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env python
+# Priority Queue Python file for Efrain, Mark and Henry
+import queue as q
+
+
+class Pq(object):
+    """Initiate priority queue"""
+
+    def __init__(self, num=3):
+        """Initiate pq with optional priority range, default is 3."""
+        for x in range(1, num+1):
+            setattr(self, 'que{}'.format(x), q.Queue())
+        self.num_queues = num
```

Comment:
Our original idea was to use a list, but we thought we weren't allowed to. 
This methodology works best for a small number of priorities and would not be a good idea for larger priority ranges. We saw it as a tradeoff for computation time sorting on insert.